docs/_docs/04_reference/config-blocks-and-attributes.md

```diff
@@ -190,6 +191,14 @@ terraform {
     run_on_error = true
   }
 
+  # Run incase of any error that occurs during apply or plan
```

Comment:
```suggestion
  # After an error occurs during apply or plan, run "echo Error Hook executed". This hook is configured so that it will run
  # after any error, with the ".*" expression.
```